### ISSUE TITLE
Submit to HA Core

### ISSUE THREAD

**Issue by cgarwood** [cgarwood/homeassistant-fullykiosk#118]
I'm in the process of submitting this component to HA Core for inclusion in 2022.9. There are a few breaking changes between this custom component and the version that will be in HA Core:

- WiFi level sensor has been removed
- Last App Start time sensor has been removed
- The `light` entity for controlling the screen has been removed and replaced with a `switch` for turning the screen on and off, and a `number` entity for controlling brightness, to better line up with HA architecture. 
- The integration domain has been changed from `fullykiosk` to `fully_kiosk` - you'll need to re-setup the integration or do some find & replace magic on HA's config files to migrate.

Still a few more PRs to submit, but I'll try to keep this updated with changes until the HA release.

**Comment by cgarwood** [comment]
HA 2022.9 beta is out with the new integration in `core` - however I didn't get a chance to PR the services to open a URL, open an app, or set a configuration parameter so those 3 services will be missing in the core component for 2022.9

**Comment by adorobis** [comment]
Great news! Do you still need some beta testing or it is ready for final release?

**Comment by chpego** [comment]
Nice job Charles !!! 🥰

**Comment by KNXBroker** [comment]
Very nice,  a lot of new integrations are pending for months, but this integration did the job within one. Beta tested without problems. Big thank you

**Comment by cgarwood** [comment]
@chpego planning to copy over the services for open url, open app, and set config parameter. I believe the rest of the services have been replaced by button entities or stock media_player services.

**Comment by DavidDeSloovere** [comment]
Thanks for moving this to core! Awesome job.
Just 'migrated' and after some device/entity id renames everything seems to be working fine.

**Comment by h4nnes** [comment]
Hi @cgarwood ,
is the Service: fullykiosk.start_application already available? I'm using this service to open a native android app.
If not, can I still use the HACS version then instead or should I stay in an older HASS version?
Thx!

**Comment by felipecrs** [comment]
But I updated HA to 2022.9, and the HACS version of Fully Kiosk is still working.

**Comment by DavidDeSloovere** [comment]
I would remove the HACS version though. Might get confusing after a while.

**Comment by felipecrs** [comment]
Of course, I'll do it as soon as the core version receives the start_application service.

**Comment by mclever** [comment]
It doesn't look like the services made it into the HA 9.0 release. When are you thinking those will make it? I used to use the fullykiosk.load_url to bring up doorbell camera feed when a person was detected.

**Comment by shred86** [comment]
Any chance you could also bring over the `fullykiosk.play_audio` service? I used this to play a custom beep sound on my wall mounted tablets when a door or window was open. I can't seem to find a way to replicate this using the HA media player services.

**Comment by ayw93** [comment]
Any chance the you can bring over the fully_kiosk.say command? I very much prefer the sound of the FK voice for announcements as opposed the Google TTS. Thank you!

**Comment by PexJensen** [comment]
What is the right/easiest way to switch from the HACS version to the new HA core version?

**Comment by DavidDeSloovere** [comment]
Delete the integration from HACS and add official one on the integration page.
You might want to check any automations for device/entity ids that might change. (off topic, check [watchman](https://github.com/dummylabs/thewatchman))

**Comment by Protoncek** [comment]
Hi guys!
I use fully kiosk on 4 of my tablets and this addon is perfect for controlling them (thanks!!). 
I have one question, though: i have two identical tablets (Lenovo M10) , so all fully-addon sensors are named, say "switch.lenovo_tab_m10_screen"  and  "switch.lenovo_tab_m10_screen_2". Is there any way to rename one of them? So that i won't end up with those annoying "_2" at the end. I searched, and didn't find it. I can rename device as such, but sensors doesn't change their's names.

**Comment by adorobis** [comment]
> 2.2 Kiosk Mode Switch

This is not available in the REST API of kiosk app so I don't think it is at all possible. As far as I was checking the integration has implemented all available features of the API. You can check here:
https://www.fully-kiosk.com/en/#rest

**Comment by adorobis** [comment]
> Or does this mean the Kiosk lock (which is indeed implemented)? A bit of a wording/function confusion here.

Yes, I believe this is the one. But those are two different things. As far as I know kiosk lock will require password to unlock but kiosk mode will still be on. It seems that you can disable kiosk mode only from the app on the device.

Regarding services: are you running latest version of HA? Maybe it was implemented later.

For volume: there is no such service dedicated to kiosk mode, probably you can use `media_player.volume_set` instead.

**Comment by adorobis** [comment]
> I'm on HA Core 2022.9.7 now.

So you need to upgrade. According to [this one](https://github.com/home-assistant/core/issues/78597) it has been added in 2022.10 release.

**Comment by adorobis** [comment]
> I can rename device as such, but sensors doesn't change their's names.

@Protoncek  you can just rename each entity_id as well

**Comment by adorobis** [comment]
Well, actually you can rename all entities of a device in one go. When you change the device name it will ask if all entities should be renamed as well. Works great:
![image](https://user-images.githubusercontent.com/19752098/205696304-2003375d-b4f4-494f-81e5-32df86cf0bee.png)
![image](https://user-images.githubusercontent.com/19752098/205696456-95491ab2-6346-4499-bac5-bda00adad587.png)

**Comment by Protoncek** [comment]
Ahhh… 
I renamed in wrong place, so HA didn’t ask me that… on basic devices page i selected my device, clicked three dots and rename.
Thanks!
